docs/.vuepress/components/CssUtilities.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div class="utility-wrapper">
+    <template v-for="(v,k) in utilityData">
+      <template v-if="nextIsCategory(v)">
+        <!-- Title -->
+        <component
+          :is="`h${level}`"
+          class="utility-class-title"
+        >{{ k }}</component>
+        <!-- Recurse -->
+        <css-utilities
+          :level="level + 1"
+          :data-path="[...dataPath, k]"
+        />
+      </template>
+
+      <template v-else>
+        <!-- Title -->
+        <component
+          :is="`h${level}`"
+          class="utility-class-title"
+        >{{ k }}</component>
+        <!-- Table -->
+        <table class="utility-table cdr-mb-space-four-x">
+          <tbody>
+            <tr v-for="(data, name) in v">
+              <td>
+                <p><b>{{ name }}</b></p>
+              </td>
+              <td>
+                <p>{{ data.description }}</p>
```

Comment:
it would be supper helpful if the description could also contain the values for each

Reply:
Values are now included below the description